src/Collections-Strings/String.class.st

```diff
@@ -1072,25 +1068,37 @@ String >> endsWith: suffix [
 { #category : #testing }
 String >> endsWithAColon [ 
 	"Answer whether the final character of the receiver is a colon"
-
+	"'displayStringOn:' endsWithAColon >>> true"
+	"'displayStringOn:foo' endsWithAColon >>> false"
+	
 	^ self notEmpty and: [ self last == $: ]
 
 ]
 
 { #category : #testing }
 String >> endsWithDigit [
 	"Answer whether the receiver's final character represents a digit."
-
+	"'foo10' endsWithDigit >>> true"
+	"'foo10foo' endsWithDigit >>> false"
+	"'foo1' endsWithDigit >>> true"
+	
 	^ self notEmpty and: [self last isDigit]
 ]
 
 { #category : #formatting }
 String >> expandMacros [
+	"'<t>' expandMacros >>> String tab"
```

Comment:
IMO, it would be good to add `"For more information about macros, check the comment of String>>#expandMacrosWithArguments:"`